pkgs/misc/uboot/default.nix

```diff
@@ -33,21 +33,26 @@ let
         url = https://github.com/dezgeg/u-boot/commit/rpi-2017-11-patch2.patch;
         sha256 = "0bbw0q027xvzvdxxvpzjajg4rm30a8mb7z74b6ma9q0l7y7bi0c4";
       })
-      (fetchpatch {
-        url = https://github.com/dezgeg/u-boot/commit/pythonpath-2017-11.patch;
-        sha256 = "162b2lglp307pzxsf9m7nnmzwxqd7xkwp5j85bm6bg1a38ngpl9v";
-      })
+      ./pythonpath-2018-01.patch
```

Comment:
I uploaded these to:

https://github.com/dezgeg/u-boot/commit/pythonpath-2018-03.patch
https://github.com/dezgeg/u-boot/commit/extlinux-path-length-2018-03.patch

so we can keep using fetchpatch.